packages/grid/_modules_/grid/components/containers/GridDataContainer.tsx

```diff
@@ -13,15 +10,11 @@ export function GridDataContainer(props: GridDataContainerProps) {
   const { className, ...other } = props;
   const apiRef = useGridApiContext();
   const dataContainerSizes = useGridSelector(apiRef!, gridDataContainerSizesSelector);
-  const scrollBar = useGridSelector(apiRef!, gridScrollBarSizeSelector);
 
   const style: any = {
     minWidth: dataContainerSizes?.width,
+    minHeight: dataContainerSizes?.height,
```

Comment:
We should be able to force the dimension. From what I understand, using min allows for the value to go beyond what should never happen (hiding root issues).

```suggestion
    // TODO remove min
    minWidth: dataContainerSizes?.width,
    minHeight: dataContainerSizes?.height,
```

Reply:
Yes. it was always "min". We can try to set the exact dimension in another PR. If it doesn't work it's because `dataContainerSizes.height` is wrong at some point.